apps/merkle_patricia_tree/lib/merkle_patricia_tree/storage_behaviour.ex

```diff
@@ -0,0 +1,28 @@
+defmodule MerklePatriciaTree.StorageBehaviour do
```

Comment:
What do you think about dropping the `Behaviour` and make it just `MerklePatriciaTree.Storage`? I don't think the specificity adds value here